meta-lmp-base/recipes-multimedia/alsa/alsa-utils/0001-alsactl-add-fallback-for-restoring-from-asound.state.patch

```diff
@@ -0,0 +1,66 @@
+From 15fa023f7c6f31e11cc65339bb979e89da4ae2c0 Mon Sep 17 00:00:00 2001
+From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
+Date: Thu, 23 Mar 2023 22:03:14 +0200
+Subject: [PATCH] alsactl: add fallback for restoring from asound.state
+
+On ostree-based disros asound.state cannot be installed into /var
+directory. Instead, this config installs into /usr/lib/alsa.
+
+Support restoring vendor-specific sound configuration from
+/usr/lib/alsa/asound.state if /var/lib/alsa/asound.state is
+unavailable.
```

Comment:
I would probably make the commit message a bit different and not necessarily as ostree specific, since the problem we have is that alsa-utils is assuming that a file which is system specific to be available in a rw folder that can be removed by the user at any point in time, causing a different system behavior (from initial flash).

What you added is support to restore the state from a read-only location when the var content is not available, which I think it is valid and we can even try to upstream.